recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 source:
     fn: jupyterlab-{{ version }}.tar.gz
     url: https://pypi.io/packages/source/j/jupyterlab/jupyterlab-{{ version }}.tar.gz
-    md5: d7a159f8ff02d94c3aee7fe5aa4dd13d
+    md5: d6cbfed779f32ae7dcea015ccbb346c9
```

Comment:
Also the new [PyPI site]( https://pypi.org/ ) has `sha256` checksums. So that might be useful for stronger validation.